cdp_scrapers/instances/seattle.py

```diff
@@ -136,9 +136,16 @@ def parse_content_uris(
         # ...
 
         # entire script tag text that has the video player setup call
-        video_script_text = soup.find(
+        video_script_block = soup.find(
             "script", text=re.compile(r"playerInstance\.setup")
-        ).string
+        )
+        if not video_script_block:
+            log.warning(
+                f"Couldn't find 'playerInstance.setup()' block on {video_page_url}.\n"
+                "seattlechannel.org may have changed their video page html"
+            )
+            return []
```

Comment:
If I remember and understand the function flow well enough, this returning `[]` (empty list) means that down the line, the event will simply be skipped over by the pipeline correct?